### PR TITLE
ovnkube-trace: Auto determine SSL status for OVN

### LINKED ISSUE #2309: ovnkube-trace: Auto determine SSL status for OVN
ovnkube-trace should not have to rely on command line flag "-noSSL". That is information which can be derived from the deployment automatically.

### PR DESCRIPTION
**Pull request by andreaskaris**
Determine SSL status from the deployment automatically instead of
relying on a command line flag.

Fixes #2309

Signed-off-by: Andreas Karis <ak.karis@gmail.com>

**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: go-controller/cmd/ovnkube-trace/ovnkube-trace.go [hunk 1]
```diff
@@ -480,6 +480,44 @@ func getPodInfo(coreclient *corev1client.CoreV1Client, restconfig *rest.Config,
...
+			klog.V(5).Infof("Cannot find ovnkube pods by label %s", nameLabel)
+			return false, err
+		}
+	}
+
+	masterPodName := pods.Items[0].Name
+	klog.V(5).Infof("Found master pod %s", masterPodName)
+
+	psCmd := "ps aux | egrep -o 'nb-address( |=)(tcp|ssl)' | awk -F ' |=' '{print $NF}'"
+	hostOutput, hostError, err := execInPod(coreclient, restconfig, ovnNamespace, masterPodName, "ovnkube-master", psCmd, "")
```
Comment (dcbw) [inline go-controller/cmd/ovnkube-trace/ovnkube-trace.go:509]: @andreaskaris maybe we should do this *always* and pull out both (a) the protocol and (b) the ports? Like make this function `getDBURIs()` or something like that.

Comment (andreaskaris) [reply go-controller/cmd/ovnkube-trace/ovnkube-trace.go:520]: Done

**Comment by dcbw** [comment]
@andreaskaris can you rebase now that the sysfs index PR has merged? Thanks!

**Comment by andreaskaris** [comment]
Found an issue, put this on hold until further notice

**Comment by andreaskaris** [comment]
Rebased.

**Pull request merged**